src/coreComponents/physicsSolvers/FieldStatisticsBase.hpp

```diff
@@ -85,11 +87,25 @@ class FieldStatisticsBase : public TaskBase
                              getDataContext(),
                              m_solverName, LvArray::system::demangleType< SOLVER >() ),
                    InputError );
+
+    // create dir for output
+    if( getLogLevel() > 0 )
```

Comment:
Here

Reply:
ok done, the reason it was done with logLevel is because I wanted to avoid rebase...